tests/factories.py

```diff
@@ -32,22 +33,24 @@ class Meta:
     name = factory.Faker("name")
 
     @factory.post_generation
-    def items(self, create, extracted, **kwargs):
+    def items(self, create, extracted, **kwargs):   # pylint: disable=method-hidden, unused-argument
+        """Creates the items list"""
         if not create:
             return
 
         if extracted:
             self.items = extracted
 
 
+
 class ItemFactory(factory.Factory):
     """Creates fake Items"""
 
     class Meta:
         model = Item
 
     id = factory.Sequence(lambda n: n)
-    shopcart_id = None
+    shopcart_id = factory.Sequence(lambda n: n)
```

Comment:
I think `shopcart_id` should not be randomly assigned. shopcart_id is always known when we're creating/updating/getting/deleting an item 🤔

Reply:
thank you for reminding this I'll modify this line and commit again